docs/Configuration.md

```diff
@@ -235,3 +235,5 @@ Here is a list of all features which can be toggled on/off.
 | `TransferAnalyzer`                     | Analyze transfers during graph build                                                                                                                                                            | no                 | yes     |
 | `FlexRouting`                          | Enable FLEX routing                                                                                                                                                                             | no                 | yes     |
 | `FloatingBike`                         | Enable floating bike routing                                                                                                                                                                    | yes                | yes     |
+| `NoTransfersOnIsolatedStops`           | If enabled do not allow any transfers either from or to NeTEx stop place with limitedUse marked as ISOLATED (only transit links are allowed)                                                    | no                 | yes     |
```

Comment:
This should not be an OTPFeature, but instead be in the netex configuration

Reply:
Drop configuration for this until it is needed - we do not need it, do you need it in Skåne?

Reply:
Yes, we need to be able to turn of this feature in case something is wrong with our data. I moved it into netex configuration.